styles/nav.scss

```diff
@@ -153,6 +153,32 @@ $daptiv-subnav-tab-h-pad: 14px;
         }
     }
 
+    .daptiv-subnav-prev-tab {
+        @extend .daptiv-subnav-tab;
+
+        &::before {
+            @extend .fa;
+            content: $fa-var-angle-double-left;
+            font-size: $icon-size-small;
+            margin-right: 3px;
+            position: relative;
+            top: 1px;
+        }
```

Comment:
agreed, but would only be nice to have for this pr. I can catch it on the semantic revision story otherwise.